test/over_react/component_declaration/builder_integration_tests/component2/constant_required_accessor_integration_test.dart

```diff
@@ -165,69 +141,67 @@ void main() {
           attachedToDocument: true,
         );
 
-        expect(consoleErrors, isEmpty, reason: 'should not have outputted a warning but found: $consoleErrors');
-
-        jacket.rerender((ComponentTest()
-          ..required = true
-          ..nullable = true
-          ..requiredAndLengthLimited = [1,2]
-        )());
-
-        expect(consoleErrors, isEmpty, reason: 'should not have outputted a warning but found: $consoleErrors');
+        expect(() {
+          jacket.rerender((ComponentTest()
+            ..required = true
+            ..nullable = true
+            ..requiredAndLengthLimited = [1,2]
+          )());
+        }, logsNoPropTypeWarnings);
       });
     });
 
     group('not throwing when a prop is nullable and set to null', () {
       test('on mount', () {
-       render((ComponentTest()
-          ..nullable = null
-          ..requiredAndLengthLimited = [1,2]
-          ..required = true
-        )());
-        expect(consoleErrors, isEmpty, reason: 'should not have outputted a warning but found: $consoleErrors');
+        expect(() {
+          mount((ComponentTest()
+            ..nullable = null
+            ..requiredAndLengthLimited = [1,2]
+            ..required = true
+          )());
+        }, logsNoPropTypeWarnings);
       });
 
       test('on re-render', () {
-        var jacket = mount((ComponentTest()
+        TestJacket jacket;
+
+        expect(() {
+          jacket = mount(
+              (ComponentTest()
+                ..required = true
+                ..nullable = true
+                ..requiredAndLengthLimited = [1,2])(),
```

Comment:
#nit component formatting
```suggestion
                ..requiredAndLengthLimited = [1,2]
              )(),
```